src/ext/java/org/opentripplanner/ext/legacygraphqlapi/LegacyGraphQLUtils.java

```diff
@@ -102,4 +102,8 @@ public static PlaceType toModel(LegacyGraphQLFilterPlaceType type) {
   public static Instant getTimeOrNow(long epochSeconds) {
     return epochSeconds != 0 ? Instant.ofEpochSecond(epochSeconds) : Instant.now();
   }
+
+  public static boolean startsWith(String str, String name, Locale locale) {
```

Comment:
How about adding a second method with the following interface

```
  public static boolean startsWith(I18NString str, String name, Locale locale) {
```

Reply:
Done